site/en/blog/new-in-chrome-87/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: New in Chrome 87
+description: >
+  Chrome 87 is rolling out now! You can now control pan, tilt, and zoom on
+  webcams that support it, range requests and service workers don’t require
+  as many workarounds, the font access API starts it’s origin trial, and
+  plenty more. Let’s dive in and see what’s new for developers in Chrome 87!
+layout: 'layouts/blog-post.njk'
+date: 2020-11-17
+updated: 2020-11-23
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/xFjaDDkGd2mBReqdPxdy.png'
+alt: >
+  New in Chrome hero logo
+---
+
+Chrome 87 is starting to roll out to stable now.
+
+{% youtube id='NCKMMzVn1c8' %}
+
+Here's what you need to know:
+
+* [Chrome Dev Summit](#cds) is back on December 9 & 10.
+* You can now control [pan, tilt, and zoom](#ptz) on webcams that support it.
+* [Range requests](#rr) and service workers don’t require as many workarounds.
+* The [font access API](#font-access) starts it’s origin trial.
+* And, there’s [plenty more](#more).
+
+I’m [Pete LePage](https://twitter.com/petele), working, and shooting
+from home, let’s dive in and see what’s new for developers in Chrome 87!
+
+## Chrome Dev Summit {: #cds }
+
+{% img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/M6GzgK34vmmwUhsymAmI.png", alt="Chrome Dev Summit logo" %}
```

Comment:
How do I get this to appear half-width, left side? I tried `className="w-figure--inline-left"` but that didn't work.